index.html

```diff
@@ -284,16 +285,19 @@ <h2 class="header-text">About <br>Myself</h2>
       </div>
     </div>
   </section>
+  <section class="contact-form">
+    <div class="contact-me">
+      <h2>Contact me</h2>
+      <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+    </div>
+    <form action="/my-handling-form-page" method="post">
+      <input class="input-1" type="text" placeholder="Yeremias"  maxlenght="30" required>
+      <input class="input-2" type="email" name="email" placeholder="natajayanj14@gmail.com" maxlength="30" required>
+      <textarea class="input-3" name="textarea" id="" cols="30" rows="10" maxlength="500" placeholder="Write your message here"></textarea>
+      <button id="btn">Get in touch</button>
+      <img class="footer-shape" src="./images/Shape.svg" alt="">
```

Comment:
As previously requested, this line is still not positioned correctly and it is making the form section look a bit weird. Kindly position it properly to introduce the orderliness of items on the screen.

Do so by:
set a position: fixed; property on the contact-form class

then add the below code to the footer img
```
position: absolute;
bottom: 4px;
left: 50%;
transform: translateX(-50%);
```

I think this might resolve the issue.